c++/src/Reader.cc

```diff
@@ -1099,6 +1137,16 @@ namespace orc {
     } while (sargsApplier_ && currentStripe_ < lastStripe_);
 
     if (currentStripe_ < lastStripe_) {
+      if (enableAsyncPrefetch_) {
+        // FIXME: this is very coarse since I/O ranges of all selected columns are about to
```

Comment:
nit. Please use IDed TODO like `TODO(ORC-XXX)` style instead of `FIXME`. It's because IDed TODOs gives us a community-wide visibility than this hidden comment.